region/maridia/inner-yellow.json

```diff
@@ -1370,7 +1370,7 @@
               ],
               "bypassStrats": [
                 {
-                  "name": "Botwoon Skip",
+                  "name": "Butterfly Room Botwoon Skip with Wall Ice Clip",
```

Comment:
I don't know that calling it Botwoon skip makes sense anymore.  You are skipping a Draygon door lock.